dojo/tools/veracode/parser.py

```diff
@@ -175,6 +175,8 @@ def __xml_static_flaw_to_finding(cls, app_id, xml_node, mitigation_text, test):
         _sast_source_obj = xml_node.attrib.get('functionprototype')
         finding.sast_source_object = _sast_source_obj if _sast_source_obj else None
 
+        finding.tags = ["sast"]
```

Comment:
```suggestion
        finding.unsaved_tags = ["sast"]
```

You have to store the tag in `unsaved_tags`, they will be written to the database after the finding has been saved.